aten/src/ATen/core/NamedRegistrations.cpp

```diff
@@ -65,6 +65,7 @@ TORCH_LIBRARY_IMPL(aten, Named, m) {
   m.impl("arccosh.out", CppFunction::makeFallthrough());
   m.impl("arccosh_", CppFunction::makeFallthrough());
   m.impl("as_strided", CppFunction::makeFallthrough());
+  m.impl("as_strided_", CppFunction::makeFallthrough());
```

Comment:
This is needed because of the fact that TensorIterator internally calls `as_strided_` when performing the cast from `common_dtype_` to `output_dtype`. I figure this is safe to add because we already have a fall through for `as_strided` above it.